### PR TITLE
CLDC-1523 buyer 2 age

### PR DESCRIPTION
**Pull request by dushan-madetech**
Adds changes needed for buyer 2 age along with accompanying tests

### CODE REVIEW

File: app/models/form/sales/questions/age2.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class Form::Sales::Questions::Age2 < ::Form::Question
+  def initialize(id, hsh, page)
+    super
+    @id = "age2"
+    @check_answer_label = "Buyer 2’s age"
+    @header = "Age"
+    @type = "numeric"
+    @page = page
+    @width = 2
```
Comment (baarkerlounger) [inline app/models/form/sales/questions/age2.rb:9]: Should this be 3?

**Review by baarkerlounger — APPROVED** [review]

File: app/models/form/sales/questions/age2.rb (hunk 1 above)
Comment (dushan-madetech) [reply app/models/form/sales/questions/age2.rb:9]: The width?

Comment (baarkerlounger) [reply app/models/form/sales/questions/age2.rb:9]: Yep? You could feasibly have a 100+ year old buyer?

Comment (dushan-madetech) [reply app/models/form/sales/questions/age2.rb:9]: I just mimicked what was set up for the lead tenant age too. I think that one should also be changed too if this one is

**Pull request merged**